converted-ethereum-tests.txt

```diff
@@ -47,3 +47,6 @@ GeneralStateTests/Cancun/stEIP1153-transientStorage/16_tloadGas.json
 GeneralStateTests/Cancun/stEIP1153-transientStorage/18_tloadAfterStore.json
 
 GeneralStateTests/Cancun/stEIP1153-transientStorage/20_oogUndoesTransientStoreInCall.json
+
+([#975](https://github.com/ethereum/execution-spec-tests/pull/975))
+src/GeneralStateTestsFiller/VMTests/vmTests/pushFiller.yml
```

Comment:
yes as you can see from other lines it expects the path to (old) filled test .json
it will be picked to run the coverage against .py generated test fixtures

Reply:
Thanks!